src/commands/account/swankyAccountCommands.ts

```diff
@@ -0,0 +1,45 @@
+import { Command } from "@oclif/core";
```

Comment:
@ipapandinas Why is that class here? Isn't it more logical to put it into `src/lib/..`? When we use `swanky account --help` oclif thinks that it is a command.

Reply:
The idea of having `SwankyAccountCommand` is to encapsulate common methods for account commands by inheriting this abstract class. But you are right we should move it, otherwise Oclif discovers it as a command.